examples/libraries/sdl2/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
put everything inside a "introduction" folder.
This way we can add more ``sdl2`` advanced examples in the future.